tests/_selftest.ts

```diff
@@ -17,7 +17,7 @@ export async function test(options: RunOptions) {
 				)
 			}
 			pkg.scripts.selftestscript =
-				"[ -d ../../vite/packages/vite/dist ] || (echo 'vite build failed' && exit 1)"
+				"pnpm vite -v || (echo 'vite build failed' && exit 1)"
```

Comment:
This is a test too, since we do not have the repo anymore in commit mode, i just tried logging the vite version. 

I think there should be a condition here.